quiz.js

```diff
@@ -25,7 +39,18 @@ if (square(2) !== 4 || cube(3) !== 27 || quad(4) !== 256) {
 // If the flag is falsy but not undefined, it should return half that sum
 
 /* your code starts here */
+function sum(numbers, twiceOrHalf) {
 
+numbers = [];
+  if (twiceOrHalf === undefined) {
```

Comment:
you need to sum up the the elements of numbers somewhere. also be careful with your conditional logic. what happens when twiceOrHalf is falsey but not undefined?